superset-frontend/src/dashboard/components/DashboardBuilder/DashboardBuilder.tsx

```diff
@@ -354,6 +361,14 @@ const DashboardBuilder: FC<DashboardBuilderProps> = () => {
     ({ dropIndicatorProps }: { dropIndicatorProps: JsonObject }) => (
       <div>
         {!hideDashboardHeader && <DashboardHeader />}
+        {nativeFiltersEnabled &&
```

Comment:
Can you extract `nativeFiltersEnabled && !editMode` to a const called `showFilterBar` to be used in both places?